### PR TITLE
aeri_irt

### PR DESCRIPTION
**Pull request by kenkehoe**
I've got a lot hear because they are all part of the AERI transition. There is a function to derive the IRT equivalent temps from AERI, new test for QC in a new file, a fix for detecting the QC variables and suppressing a warning in utility function.

### CODE REVIEW

**Comment by AdamTheisen** [comment]
Thanks @kenkehoe !  Will look through this today.  Failing test is 

./act/retrievals/aeri.py:266:40: E128 continuation line under-indented for visual indent

**Comment by AdamTheisen** [comment]
@kenkehoe First question, what is qctests2 and how is it different from qctests?  Should we combine or make qctests2 more specific?

File: act/retrievals/planck.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+
+def planck(wnum=None, radiance=None, temperature=None):
```
Comment (AdamTheisen) [inline act/retrievals/planck.py]: do we want to rename this to something else more descriptive of what it's doing?  right now we're calling planck.planck()

File: act/retrievals/planck.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    # h = Plancks's constant
+    # k = Boltzmann's constant
+    # c = Speed of light in a vacuum
+    # C1 = 2 h c^2, [W cm^2]
+    C1 = 1.191066e-5  # For radiance in units of mW m^-2 sr^-1 /cm^-1
+    # C2 = h c / k, [K cm]
+    C2 = 1.438833
+
+    if radiance is not None:
```
Comment (AdamTheisen) [inline act/retrievals/planck.py]: We also might want to consider adding in some error messages.  I can call this function without providing any input and it seems to work fine... just no output....

**Comment by AdamTheisen** [comment]
aeri_vs_irt
Testing to ensure AERI compares with IRT data.  Brought to light that the AERI and IRT do not compare well in cold conditions.  I added information from the handbook into the aeri2irt description.

**Comment by AdamTheisen** [comment]
Note to self... Checked Planck function results against NOAA calculator and numbers verified.

https://ncc.nesdis.noaa.gov/data/planck.html

**Comment by AdamTheisen** [comment]
@kenkehoe I know I have a lot of comments in here, but to summarize

I pushed a commit to aeri.py to resolve the pep8 issue, add some additional information in the docstring description, and remove an unused import.

**questions for you**

what is qctests2 and how is it different from qctests?  Should we combine or be more specific?

Do we want to rename the planck function to something more descriptive?  Right now the call is act.retrievals.planck.planck()

We should shore up the planck function as I can call it without passing anything and it does not error.  

Everything else looks good and verifies well!

**Comment by kenkehoe** [comment]
- what is qctests2 and how is it different from qctests? Should we combine or be more specific?

I have set up the QC Tests class to all use the qcfilter class but spread the code across multiple files. The hope is to keep the files to a reasonable size, like under 1000 lines. That was what I found online as a suggestion. The qctest file is the basic stuff with simple limits. qctest2 is my fist more complex test. The plan was to keep adding more tests in qctest2 until we get to about 1000 lines and then add qctest3,...

What do you think about that logic?

**Comment by kenkehoe** [comment]
- Do we want to rename the planck function to something more descriptive? Right now the call is act.retrievals.planck.planck()

Sure better naming is always my goal. I'll look into updating this.


- We should shore up the planck function as I can call it without passing anything and it does not error.

OK, I'll look into this. I was using the logic to mimic unix utility stye programming where if asked to do nothing it does not crash. Let me see what I can do to improve this.

**Comment by AdamTheisen** [comment]
@rcjackson @scollis  what are your thoughts on the qctests/qctests2/qctests3 item?  I think this is reasonable, but wanted to get other thoughts.  

> * what is qctests2 and how is it different from qctests? Should we combine or be more specific?
> 
> I have set up the QC Tests class to all use the qcfilter class but spread the code across multiple files. The hope is to keep the files to a reasonable size, like under 1000 lines. That was what I found online as a suggestion. The qctest file is the basic stuff with simple limits. qctest2 is my fist more complex test. The plan was to keep adding more tests in qctest2 until we get to about 1000 lines and then add qctest3,...
> 
> What do you think about that logic?

**Comment by rcjackson** [comment]
As long as we document in the qctests* section why we are doing 1, 2, 3 (i.e. Level 1, 2, 3 QC) then I see that making sense.

**Comment by kenkehoe** [comment]
I pushed my changes but I don't see Travis CI running?

**Comment by AdamTheisen** [comment]
It does look like the Travis-CI ran 17 hours ago which coincides with your comment.  I'll see if I can rerun it just to make sure

**Comment by AdamTheisen** [comment]
All checks pass, retrieval compares well and planck function is looking good.  Merging

**Pull request merged**